locale/af.po

```diff
@@ -13739,7 +13739,7 @@ msgid "&Generate"
 msgstr "&Genereer"
 
 #: src/menus/PluginMenus.cpp
-msgid "Add / Remove Plug-ins..."
+msgid "Plugin Manager"
```

Comment:
@Paul-Licameli is it necessary to update po files each time new msgid added or removed from the sources?